.gitlab/source_test/macos.yml

```diff
@@ -22,7 +22,7 @@ tests_macos:
     - python3 -m pip install -r tasks/libs/requirements-github.txt --break-system-packages
     - FAST_TESTS_FLAG=""
     - if [[ "$FAST_TESTS" = "true" ]]; then FAST_TESTS_FLAG="--fast-tests true"; fi
-    - inv -e github.trigger-macos --workflow-type "test" --datadog-agent-ref "$CI_COMMIT_SHA" --version-cache "$VERSION_CACHE_CONTENT" $FAST_TESTS_FLAG --test-washer $COVERAGE_CACHE_FLAG
```

Comment:
This variable is not meant for this task, but for [coverage.upload-to-codecov](https://github.com/DataDog/datadog-agent/blob/main/.gitlab/source_test/common.yml#L8)